lib/util.js

```diff
@@ -191,59 +192,47 @@ function getLink(obj, linkName, altName) {
   } else {
     return link;
   }
-}
+};
+
+util.getNativeConsole = function() {
```

Comment:
Am curious how often we call console (guessing not often) - it's probably okay this way, but might be nicer to move this to a singleton type pattern where we're only making this check once.